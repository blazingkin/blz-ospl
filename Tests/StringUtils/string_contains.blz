print("asdf".contains?("a"))
print("asdf".contains?(""))
print("adsf".contains?("g"))
print("asdf".contains?("asdf"))
print("asdf".contains?("asdfuiop"))
print("asdf".contains?("dfasdf"))