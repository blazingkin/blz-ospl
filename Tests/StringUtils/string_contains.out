true
true
false
true
false
false
